FirstApp/ViewControllers/HomeViewController.swift

```diff
@@ -1,7 +1,50 @@
 import UIKit
 
 class HomeViewController: UIViewController {
+
+    var name = ["mon4", "mon5", "mon6", "mon7", "mon8", "mon9", "mon10", "cherry", "apples"]
```

Comment:
What is this for ? Was it dummy data ? Should delete it if you not using.

Reply:
Yes it is dummy data😌. I have now deleted it. Thank you.